CenterForOpenScience/ember-osf-web
# [ENG-668] Allow meeting submissions to be sorted by download count

- Ticket: [ENG-668]
- Feature flag: n/a

## Purpose
- Allow meeting submissions to be sorted by download count

## Summary of Changes
- Add sort buttons to download column of the meetings detail page (osf.io/meetings/<meeting_id> )
- We can turn on `ember_meeting_detail_page` on production after this is merged

## Side Effects
None

## QA Notes
- Please verify that meeting submissions are sorting by download count, and that the sorting is working between submission pages

[ENG-668]: https://openscience.atlassian.net/browse/ENG-668
